src/components/__tests__/LibraryStageItem-test.tsx

```diff
@@ -40,6 +40,11 @@ describe("LibraryStageItem", () => {
     expect(options.at(1).html()).to.contain("Production");
     expect(options.at(2).html()).to.contain("Cancelled");
   });
+  it("should have an aria-label", () => {
+    let select = wrapper.find("select");
+
+    expect(select.prop("aria-label")).to.equal(`Select ${label}`);
```

Comment:
Unnecessary blank line (or is it just being rendered weirdly on Github?).  Looks good otherwise!

Reply:
That was intentional as I like a link after variables and before the tests but I can remove.

Reply:
I usually like that too, but it looks weird to me because the test is only two lines long.  Not a big deal either way, though.